packages/twenty-server/src/engine/api/graphql/graphql-query-runner/helpers/process-aggregate.helper.ts

```diff
@@ -19,17 +15,18 @@ export class ProcessAggregateHelper {
     for (const [aggregatedFieldName, aggregatedField] of Object.entries(
       selectedAggregatedFields,
     )) {
-      const fieldMetadata = fieldMetadataMapByName[aggregatedField.fromField];
-
-      if (!fieldMetadata) {
+      if (
+        !aggregatedField?.fromColumnName ||
```

Comment:
I'm not a big fan of these tests not being done on boolean.
I would prefer gate such as isUndefined

Reply:
Sure